src/Pages/Home/Home.js

```diff
@@ -0,0 +1,16 @@
+import {Header} from "../../components/Header/Header"
+import {Category} from "../../components/Category/Category"
+import {Baner} from "../../components/Baner/Baner"
+import {Footer} from "../../components/Footer/Footer"
+
+const Home=()=>{
+    return(
+    <div>
```

Comment:
You have made used of fragments in some places so you can alse replace this <div> here with the same to maintain the consistency

Reply:
ok